server/src/logger/opts.ts

```diff
@@ -0,0 +1,8 @@
+const SimpleNodeLogger = require('simple-node-logger'),
+  opts = {
+    logFilePath: 'mylogfile.log',
```

Comment:
jeśli chodzi o ścieżkę to proponuje zrobić w folderze server folder logs i dodac to do .gitignore